test/unittests/test_excel2xml.py

```diff
@@ -527,8 +527,8 @@ def test_excel2xml(self) -> None:
             with open("excel2xml-output-data.xml") as f:
                 returned = f.read()
                 self.assertEqual(returned, expected, msg=f"Failed with extension {ext}")
-        if os.path.isfile("excel2xml-output-data.xml"):
-            os.remove("excel2xml-output-data.xml")
+            if os.path.isfile("excel2xml-output-data.xml"):
```

Comment:
Indentation changed by purpose - the artifact should be deleted immediately after every cycle of the loop.